docs/tableSonar.adoc

```diff
@@ -9,17 +9,17 @@ s|Group attributes | |
  |{attr}:sonar_serial_number |R |Sonar serial number.
  |{attr}:sonar_software_name |R |Sonar software name.
  |{attr}:sonar_software_version |R |Sonar software version.
- |{attr}:sonar_type = "omni-sonar" |M |Type of sonar, chosen from a defined vocabulary (currently only one value): ”omnisonar” (a sonar with a nominally omnidirectional mode).
+ |{attr}:sonar_type = "omni-sonar" |M |Type of sonar, chosen from a defined vocabulary: "omnisonar" (a sonar with a nominally omnidirectional mode), and "echosounder" (a sonar with a single directed beam).
  
 s|Types | |
  2+|{var}byte enum beam_stabilisation_t {not_stabilised = 0, stabilised = 1} |Whether or not the beam direction is compensated for platform motion.
  2+|{var}byte enum beam_t {single = 0, split_aperture_angles = 1, split_aperture_4_subbeams = 2, split_aperture_3_subbeams = 3, split_aperture_3_1_subbeams = 4} |Beam type. Split aperture indicates a beam that can detect the arrival angle of echoes, while single beam cannot.
- 2+|{var}byte enum conversion_equation_t {type_1 = 1, type_2 = 2, type_3 = 3, type_4 = 4, type_5 = 5} |The type of equation used to convert backscatter measurements into volume backscattering and target strength.
+ 2+|{var}byte enum conversion_equation_t {type_1 = 1, type_2 = 2, type_3 = 3, type_4 = 4, type_5 = 5, type_6 = 6} |The type of equation used to convert backscatter measurements into volume backscattering and target strength.
  2+|{var}byte enum transmit_t {CW = 0, LFM = 1, HFM = 2} |Type of transmit pulse. CW = continuous wave – a pulse nominally of one frequency, LFM = linear frequency modulation – a pulse which varies from transmit_frequency_start to transmit_frequency_stop in a linear manner over the nominal pulse duration (transmit_duration_nominal), HFM = hyperbolic frequency modulation – a pulse which varies from transmit_frequency_start to transmit_frequency_stop in a hyperbolic manner over the nominal pulse duration (transmit_duration_nominal).
 
 s|Subgroups | |
  |{var}Beam_group1 |M |Example of a beam group. Include as many subgroups as necessary for different beam groups. Use unique group names, preferably of the form Beam_groupX where X is an integer.
```

Comment:
Is it allowed to have data with grid_group but no beam_group ? ie data containing only gridded values 
In such a case both Beam_group and Grid_group should be optionnals

Reply:
I can see valid reasons for a file with a grid group and no beam group, so agree that they can both be optional.

Reply:
I have added a commit to this PR that makes beam_group optional.